### ISSUE TITLE
measurement_list functions return lowercase results

### ISSUE THREAD

**Issue by jeffcnz** [mullenkamp/hilltop-py#62]
measurement_list functions in web_service and Hilltop class now convert MeasurementNames to lower case (they didn''t use to).  This can cause issues with downstream processes if they are case sensitive.

Please could the option for lowercase, or provided case be added.

**Comment by mullenkamp** [comment]
Thanks for bringing this up. 

When I was reworking the python package I was trying to better represent Hilltop's data model. This meant adding the Data Source Name as part of the requested Measurement Name. Unfortunately as I learned through that process, Hilltop is not consistent in a variety of ways. Part of the learning process was that Hilltop is not case sensitive for requests using the Measurement Name and the Data Source Name, and I had to make them lowercase to deal with a particular issue with Data Source Names that sometimes returned as upper or lowercase from Hilltop. I've ultimately ditched using the Data Source Name entirely (as it caused more issues than it was solving), but I didn't revert the lowercase Measurement Names back to whatever Hilltop returns as the "RequestAs" (which is what Hilltop-py defines as the Measurement Name for requesting data).

I will have the Measurement Name returned as exactly what Hilltop returns as the "RequestAs" with the cases in tact. 

And just an FYI, the "RequestAs" may include the Data Source Name in brackets if there are multiple Measurement Names with the same name and it is not labeled as the default Measurement Name. For the intents and purposes of requesting data from Hilltop, this is as good as it gets...for better or worse...
